### PR TITLE
[Forwardport]Fix issue port 19328 - Success Message Icon vertically misaligned in admin panel

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Original Pull Request 
magento/magento2#19333

### Description (*)

Success Message icon misaligned.
### Fixed Issues (if relevant)

1. magento/magento2#19328 : Success Message Icon vertically misaligned in admin panel

### Manual testing scenarios (*)

1.Login to admin panel > System > Import.
2.Select necessary Option > Upload CSV.
3.Click on Check Data Button > Check success Message Icon

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3524](https://jira.corp.magento.com/browse/ENGCOM-3524) has been created to process this Pull Request

File: app/design/adminhtml/Magento/backend/web/css/source/components/_messages.less [hunk 1]
```diff
@@ -121,6 +121,15 @@
     }
 }
 
+#import_validation_container  {
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/source/components/_messages.less]: @speedy008 can you please change the selector to avoid using the ID

Comment (SyntaxKay) [reply app/design/adminhtml/Magento/backend/web/css/source/components/_messages.less]: Hi @sivaschenko 
I have Updated code.

Comment (sivaschenko) [reply app/design/adminhtml/Magento/backend/web/css/source/components/_messages.less]: Thank you!

**Comment by sivaschenko** [comment]
Hi @speedy008 Success Message Icon is vertically misaligned with phrase "Import successfully done"
image

**Comment by SyntaxKay** [comment]
@sivaschenko Can you explain whats the issue
As i have observe the Success Message Icon is vertically misaligned is already has been fixed but import icon still remain resolved

**Comment by sivaschenko** [comment]
Hi @speedy008 do you mean that only verification success message icon has been fixed and not the icon shown after successful import? Is there a universal solution that can fix both icons?

**Comment by SyntaxKay** [comment]
@sivaschenko I have create one common solution for vertically icon middle issue.

**Comment by sivaschenko** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, here is your [new Magento instance](https://pr-19334.instances.magento-community.engineering).
Admin access: [https://pr-19334.instances.magento-community.engineering/admin](https://pr-19334.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**